docs/proposals/network-policy.md

```diff
@@ -176,14 +217,12 @@ def is_traffic_allowed(traffic, pod):
              ingress_rule.ports.matches(traffic):
           return True 
 
-  # network-isolation is True and no policies match the given pod and traffic.
+  # Ingress isolation is DefaultDeny and no policies match the given pod and traffic.
   return False
 ```
 
 ### Open Questions
 
-> The question then remains whether one NP object per "app" makes sense (in this case app={a,b,c}) or whether one NP object per selector
-
 - A single podSelector per NetworkPolicy may lead to managing a large number of NetworkPolicy objects, each of which is small and easy to understand on its own.  Allowing a podSelecor per ingress rule would allow for fewer, larger NetworkPolicy objects that may be easier to manage, but harder to understand individually.  This proposal has opted to favor a larger number of smaller objects that are easier to understand.  Is this the right decision? 
```

Comment:
Can we formulate this differently? The question is whether to optionally allow for a podSelector per rule to allow the user to define the number of objects and object size. Something like this:

A single podSelector per NetworkPolicy may lead to managing a large number of NetworkPolicy objects, each of which is small and easy to understand on its own. However, this may lead for a policy change to require touching several policy objects. Allowing an optional podSelecor per ingress rule additionally to the podSelector per NP object would allow the user to group rules into logical segments and define size/complexity ratio where it makes sense. This may lead to a smaller number of objects with more complexity if the user opts in to the additional podSelector. This proposal has opted to favor a larger number of smaller objects that are easier to understand.  Is this the right decision?